monoio/src/driver/op/connect.rs

```diff
@@ -81,18 +81,18 @@ impl OpAble for Connect {
                 std::ptr::null_mut(),
             )) {
                 Err(err) if err.raw_os_error() != Some(libc::EINPROGRESS) => Err(err),
-                _ => Ok(self.fd.raw_fd() as u32),
+                _ => Ok(MaybeFd::zero()),
```

Comment:
Is this ok to return MaybeFd::zero()? it seems to change the original logic.

Reply:
The connect op is like connect syscall. If the connect is executed successfully, it returns 0. Here we don't have to return the fd since the socket has already been created.